src/unix/linux-core.c

```diff
@@ -983,3 +983,48 @@ void uv__set_process_title(const char* title) {
   prctl(PR_SET_NAME, title);  /* Only copies first 16 characters. */
 #endif
 }
+
+int uv__enable_largesocket(uv_stream_t *stream) {
+  int rmem;
+  int wmem;
+  int rs;
+  int ws;
+  int rr;
+  int rw;
+  FILE *fp = uv__open_file("/proc/sys/net/core/rmem_max");
+  if (fp == NULL)
+    return UV_ENOSYS;
+
+  rs = 0;
+  ws = 0;
+  rmem = 0;
+  wmem = 0;
+  if (fscanf(fp, "%d", &rmem) == 1)
+    rs = 1;
+  fclose(fp);
```

Comment:
I don't think `rs` is needed (or `ws`), you can return with error is `fscanf() != 1` and then just check `rmem(or `wmem`)` value.